### PR TITLE
printing comp_f.symbol variable for better logging

### PR DESCRIPTION
**Pull request by migui3230**
let me know if you have any feedback!

### CODE REVIEW

File: lang/syn/src/idl/file.rs [hunk 1]
```diff
@@ -513,6 +513,7 @@ fn idl_accounts(
         .iter()
         .map(|acc: &AccountField| match acc {
             AccountField::CompositeField(comp_f) => {
+                println!(comp_f.symbol);
                 let accs_strct = global_accs
                     .get(&comp_f.symbol)
                     .expect("Could not resolve Accounts symbol");
```
Comment (paul-schaaf) [inline lang/syn/src/idl/file.rs:519]: With this implementation we would always log the symbol which is not what armani meant. We should only log it inside the `.expect` call.

Comment (migui3230) [reply lang/syn/src/idl/file.rs:519]: @paul-schaaf Oh got it, thanks for the feedback!

**Comment by migui3230** [comment]
@paul-schaaf changes should be good now

File: lang/syn/src/idl/file.rs [hunk 2]
```diff
@@ -515,7 +515,7 @@ fn idl_accounts(
             AccountField::CompositeField(comp_f) => {
                 let accs_strct = global_accs
                     .get(&comp_f.symbol)
-                    .expect("Could not resolve Accounts symbol");
+                    .expect(format!("Could not resolve Accounts symbol {}", comp_f.symbol).as_str());
```
Comment (fanatid) [inline lang/syn/src/idl/file.rs:518]:
```suggestion
                    .unwrap_or_else(|_| panic!("Could not resolve Accounts symbol {}", comp_f.symbol));
```

**Review by fanatid — CHANGES_REQUESTED** [review]

File: lang/syn/src/idl/file.rs (hunk 2 above)
Comment (migui3230) [reply lang/syn/src/idl/file.rs]: just pushed those changes!

Comment (fanatid) [reply lang/syn/src/idl/file.rs]: Sorry, should be:
```rust
.unwrap_or_else(|| panic!
```
not
```rust
.unwrap_or_else(|_| panic!
```

Comment (migui3230) [reply lang/syn/src/idl/file.rs]: pushed those changes again! ^

**Review by fanatid — APPROVED** [review]

**Comment by paul-schaaf** [comment]
thank you!

**Pull request merged**